vmdb/app/models/miq_provision_task.rb

```diff
@@ -1,6 +1,7 @@
 class MiqProvisionTask < MiqRequestTask
   include MiqProvisionQuotaMixin
   include ReportableMixin
+  include_concern 'Tagging'
```

Comment:
The include_concern was moved here so needs to be removed from miq_provision.rb.